package.json

```diff
@@ -21,7 +21,7 @@
   },
   "type": "module",
   "scripts": {
-    "build": "tsc -p . && node scripts/copy-files.js",
+    "build": "tsc -p . && node scripts/write-all-commands.js && node scripts/copy-files.js",
```

Comment:
Just checking: is this the correct order? It is right? As in: copy-files should not come before write-all-commands.

Reply:
It doesn't matter because we're not executing the commands